.gitignore

```diff
@@ -40,3 +40,4 @@ coverage/
 ######################################
 !.gitignore
 !.gitkeep
+vendor
```

Comment:
This should go before the last two lines. Maybe under the bundler section?
I'm curious, what was creating this on your machine?  Usually this is only created in the ci.